hikari/impl/interaction_server.py

```diff
@@ -165,6 +169,15 @@ def __init__(
         rest_client: rest_api.RESTClient,
         public_key: typing.Optional[bytes] = None,
     ) -> None:
+        # This is kept inline as pynacl is an optional dependency.
+        try:
+            from nacl import signing
+
+        except ModuleNotFoundError as exc:
+            raise RuntimeError(
+                "You must install the optional `hikari[server]` dependencies to use the default interaction server."
+            ) from exc
```

Comment:
Same as the other comment

Reply:
Oh yeah another reason why I prefer this style is that we're not erasing the source error since silently throwing away the raised ModuleNotFoundError instead of using it as the cause would make it take more difficult/effort to debug any cases where it wasn't importing despite being supposedly installed (e.g. one of its own dependencies is no-longer installed).